examples/time/time.bal

```diff
@@ -14,7 +14,7 @@ public function main() returns error? {
     io:println(`Current timestamp nanoseconds as a
     fraction: ${currentUtc[1]}s`);
 
-    // Return seconds from some unspecified epoch. The returned `seconds` has
+    // Returns seconds from some unspecified epoch. The returned `seconds` has
     // the nanoseconds precision which represents using the fractional part.
```

Comment:
```suggestion
    // the nanoseconds precision, which represents using the fractional part.
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383